### ISSUE TITLE
Did you reproduce megaface ?

### ISSUE THREAD

**Issue by twmht** [TreB1eN/InsightFace_Pytorch#29]
Hi

Did you reproduce megaface result on pytorch?

thank you

**Comment by TreB1eN** [comment]
no, you can have a try

**Comment by twmht** [comment]
@TreB1eN 

I tried mobilenet, but can't reproduce the paper's result (92%)

My result is only 87%

**Comment by twmht** [comment]
@TreB1eN 

I did not train on megaface, I trained on refine MS-1M and test on megaface. Did you try to do that?

**Comment by ruiming46zrm** [comment]
> @TreB1eN
> 
> I tried mobilenet, but can't reproduce the paper's result (92%)
> 
> My result is only 87%

can you tell us your acc on cfp-fp , mine is 87.5%

**Comment by ruiming46zrm** [comment]
> 0.9504 CFP-FP(%) by IR-SE50, pretrained model are provided

training refined-ms1m ,I can get LFW 99.52% but can't get cfp-fp so well , maybe something wrong.

**Comment by twmht** [comment]
@TreB1eN 

Did you trained on refined-ms1m on IR-SE50?

**Comment by twmht** [comment]
@ruiming46zrm 

Did you use this code without modification?  I haven't tested cfp-fp yet.

**Comment by TreB1eN** [comment]
@twmht yes

**Comment by ruiming46zrm** [comment]
@twmht  yes, ok

**Comment by twmht** [comment]
@TreB1eN 

I can help to test the provided model in megaface.

**Comment by ruiming46zrm** [comment]
@twmht 
will you share mega test code if it's convenient~ ?

**Comment by twmht** [comment]
@ruiming46zrm 

most of testing code is from insightface. Did you tried to reproduce the result on insightface?

**Comment by ruiming46zrm** [comment]
@twmht  thanks, I don't know much about the megaface, I'm gonna try.

**Comment by twmht** [comment]
@TreB1eN 

in addition, mobilenet only got 74% on megaface.

**Comment by yangyuke001** [comment]
@twmht  will you share mega test code if it's convenient~ ? mainly gen_megaface.py  for pytorch version,thanks a lot

**Comment by ruiming46zrm** [comment]
> yangyuke001 At 2019-01-07 08:23:09, "ruiming46zrm" <notifications@github.com> wrote: @yangyuke001 I have one , what's your weichat? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or mute the thread.

user name unexist , mine is : ruiming46zrm

**Comment by cvtower** [comment]
Hello @ruiming46zrm, @twmht 

The mega test code has to load a trained model and do inference, i mean it can not be tested during training, right?
Then i have a question, this repo saves model per few iters, however, the best accuracy on different dataset may not always the same saved model. Shall we just take the best accuracy on each dataset as the final performance?

**Comment by edward0909** [comment]
@ruiming46zrm @yangyuke001 
Could you send me your test code on megaface @ thanh1705@hotmail.com? Thank you so much!

**Comment by ghost** [comment]
Hi,
Do we need to normalize the  images of Facescrub and Megaface before feeding them to the model to get features? Thank you for reading.

**Comment by gaow0007** [comment]
> @TreB1eN and @ruiming46zrm
> 
> I just tried @TreB1eN 's provided resnet50 pretrained weight, but only got 90% on megaface.
> 
> but from deepinsight (https://github.com/deepinsight/insightface), they provide resnet50 which can get 97.64% on megaface.
> 
> since this is a huge gap, there might be some issues with pytorch implementation.

Could you share the test configs of MegaFace? Such as select how many images as galleryset?

**Comment by HulkMaker** [comment]
> > @TreB1eN and @ruiming46zrm
> > I just tried @TreB1eN 's provided resnet50 pretrained weight, but only got 90% on megaface.
> > but from deepinsight (https://github.com/deepinsight/insightface), they provide resnet50 which can get 97.64% on megaface.
> > since this is a huge gap, there might be some issues with pytorch implementation.
> 
> Could you share the test configs of MegaFace? Such as select how many images as galleryset?

Could you share Megaface dataset since its stopping supported in their website. And I would like to test on it too. Thanks!
